time/CalendarWidget.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  CalendarWidget.swift
+//  seximal
+//
+//  Created by Henrik Storch on 24.06.21.
+//
+
+import SwiftUI
+import Intents
+import WidgetKit
+
+
+struct CalendarWidget: Widget {
+    let kind = "cal"
```

Comment:
should probably be `static` jic they're needed elsewhere and better performance(?)